srcpkgs/wtplan/template

```diff
@@ -1,19 +1,28 @@
 # Template file for 'wtplan'
 pkgname=wtplan
 version=0.1
-revision=1
-nocross="runtime/cgo: armv7l-linux-musleabihf-gcc: error: unrecognized command line option '-m64"
+revision=2
 build_style=gnu-makefile
 hostmakedepends="go"
-nopie="golang"
-nostrip="golang"
 short_desc="Calendar program with terminal and web interfaces and git integration"
 maintainer="Toyam Cox <Vaelatern@voidlinux.org>"
 license="MIT"
 homepage="https://github.com/kjellwinblad/wtplan/"
 distfiles="https://github.com/kjellwinblad/wtplan/archive/${version}.tar.gz"
 checksum=31c66181f815889828eb2bd3739d6b23582ec75a5a8d6c6bf76c2a6679112bb4
 
+do_build() {
+	export GOPATH=$(pwd)
+	export VERSIONSTR=$version
+	go generate wtplan
+	go generate wtplan-web
+	. $XBPS_COMMONDIR/environment/build-style/go.sh
+	GOPATH=$(pwd)
+	# Without -buildmode=pie, build failure on armv6hf-musl
+	go build -buildmode=pie wtplan
+	go build -buildmode=pie wtplan-web
```

Comment:
Could we consider adding these flags to the go build style?

Reply:
Looks like a yes, all architecture that our go supports also has `-buildmode=pie` on.
I'll look into drafting a change.